spec/compiler/normalize/select_spec.cr

```diff
@@ -0,0 +1,68 @@
+require "../../spec_helper"
+
+describe "Normalize: case" do
+  it "normalizes select with call" do
+    assert_expand "select; when foo; body; when bar; baz; end", <<-CODE
+      __temp_1, __temp_2 = ::Channel.select({foo_select_action, bar_select_action})
```

Comment:
_for_select or _action, which is it? :)

Not too sure a about either name yet, have to sleep over it.